test/scripts/e2e_client_runner.py

```diff
@@ -174,8 +180,8 @@ def _connect(self):
         if self.algod and self.kmd:
             return
         # should run from inside self.lock
-        xrun(['goal', 'kmd', 'start', '-t', '200'], env=self.env, timeout=5)
         algodata = self.env['ALGORAND_DATA']
+        xrun(['goal', 'kmd', 'start', '-t', '3600','-d', algodata], env=self.env, timeout=5)
```

Comment:
looks to high

Reply:
yes, that's why I added the kmd shutdown.